metricbeat/docs/modules/http.asciidoc

```diff
@@ -5,8 +5,8 @@ This file is generated! See scripts/docs_collector.py
 [[metricbeat-module-http]]
 == http Module
```

Comment:
I would use `HTTP` for the protocol, and `http` for the module name. What about `http module`?